web-animations-1/Overview.bs

```diff
@@ -2617,9 +2617,21 @@ Animation effects {#animation-effects}
   which is a signed offset from the [=start time=] of the [=animation=].
 
   The length of the [=active interval=] is called the <dfn>active duration</dfn>,
-  the calculation of which is defined in [[#calculating-the-active-duration]].
+  and is determined by the [=iteration duration=] and [=iteration count=].
```

Comment:
Should this somehow refer to the below equation rather than it just appearing unannounced? e.g.

"...and is determined by the iteration duration and iteration count as defined below:"